### DISCUSSION TITLE
Enable just the services, not the other features

### DISCUSSION THREAD

**Discussion by rschaeuble** [frenck/spook#464] [Ideas] [answered]
I'm very interested in some of Spook's services (etc. removing orphaned entities). However, I do not want the additional entities and other features that the extension provides.

I think it would be awesome being able to individually enable the extension's features when activating it.

**Comment by frenck** [comment, marked as answer]
Home assistant has features for that already. You can disable any device or entity provided by an integration.


Adding that to spook would be a duplicate feature.

../Frenck
